index.html

```diff
@@ -5,12 +5,81 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Open+Sans:300,400,600,700&amp;lang=en" />
```

Comment:
Here you are importing "Open Sans" from Google Fonts, but aren't using the font anywhere in the page. Did you mean to import "Dosis", which is used in your stylesheet?